src/Converter/textureConverter.js

```diff
@@ -27,7 +27,7 @@ export default {
                 new THREE.Color(backgroundLayer.paint['background-color']) :
                 undefined;
 
-            extentDestination.as(CRS.formatToEPSG(layer.crs), extentTexture);
+            extentDestination.toExtent(CRS.formatToEPSG(layer.crs), extentTexture);
```

Comment:
shouldn't we name it destinationTile instead for more clarity?

Reply:
Yup agree.